src/components/SponsorsSection/SponsorsSection.scss

```diff
@@ -5,3 +5,11 @@
   justify-content: space-around;
   padding: 64px;
 }
+
+@media only screen and (max-width: 860px) {
+  .sponsors-section {
+    flex-direction: column;
+    justify-content: space-between;
+    height: 375px;
```

Comment:
I added a third logo, so it spills out of this section now. I think just removing the height would fix that. I also think there should be more spacing between the logos when they're vertical (maybe why you added the height). We could add a padding to the <a>'s in the sponsor section since they work as a container for the images.

Reply:
Adding padding into .sponsors-section wont work as it'll add padding around the sponsors not in between them. I've made a change to add padding on .sponsors-section > a tag. This will give them the padding needed without ever adjusting the height in case we add more sponsors.